docs/team/godjuansan.md

```diff
@@ -0,0 +1,62 @@
+  * Highlights: User can comfortably merge two notes into one fomr the CLI.
+
+* **New Feature**: Give tags to note.
+  * What it does: Allows the user to give multiple tags to a single note.
+  * Justification: Tags give user more flexibility when storing notes and opens more possibilities of features in the future. For instance, search with tags is a useful feature that we implement after finishing this tagging feature.
+  * Highlights: User can managed their notes easily by putting tags on them.
+
+* **New Feature**: Mark a note as done or not done yet.
+  * What it does: Allows the user to mark certain notes as done and differentiate them from notes that are not done yet.
+  * Justification: The user can easily remember whether they have done a certain notes or have not done it yet.
+  * Highlights: With just a single one-liner CLI command, user can remember which notes is already done very easily.
+
+* **New Feature**: Convert note into .txt file.
+  * What it does: Convert a note that is stored in our local storage application into a .txt file.
+  * Justification: User might want to export their notes somewhere else and rather than having to manually copy and paste everything, this feature can be useful.
+  * Highlights: User can just type a one-liner command and will be able to get a .txt file rather than manually copy and paste everything.
+  
+**Code contributed**: [RepoSense link] (https://nus-cs2103-ay2021s2.github.io/tp-dashboard/?search=w13&sort=groupTitle&sortWithin=title&since=2021-02-19&timeframe=commit&mergegroup=&groupSelect=groupByRepos&breakdown=false&tabOpen=true&tabType=authorship&tabAuthor=godjuansan&tabRepo=AY2021S2-CS2103T-W13-1%2Ftp%5Bmaster%5D&authorshipIsMergeGroup=false&authorshipFileTypes=docs~functional-code~test-code&authorshipIsBinaryFileTypeChecked=false)
+
+* **Project management**:
+  * Managed all of notes feature and divide tasks between myself and jessen11.
+  
+* **Documentation**:
+  * User Guide:
+    * Added documentation for the features `markasundonenote` and `markallasundonenote` command
+      (Pull requests [\#130](https://github.com/AY2021S2-CS2103T-W13-1/tp/pull/130))
+    * Updated documentation for features `addnote` command
+      (Pull requests [\#240](https://github.com/AY2021S2-CS2103T-W13-1/tp/pull/240))
+  * Developer Guide:
+    * Added Developer Guide for markasundonenote and markallasundonenote 
+      (Pull requests [\#130](https://github.com/AY2021S2-CS2103T-W13-1/tp/pull/130))
+    * Added Logic Component Diagram
+      (Pull requests [\#142](https://github.com/AY2021S2-CS2103T-W13-1/tp/pull/142))
+    * Added Developer Guide for Merge Note
+      (Pull requests [\#174](https://github.com/AY2021S2-CS2103T-W13-1/tp/pull/174))
+
+---to be added
+* **Community**:
+  * PRs reviewed (with non-trivial review comments): [\#88]().
+  * Contributed to forum discussions: [\#279](), [\#293](), [\#295](), [\#298]().
```

Comment:
Do you forget to add the link? or is this intentional? Also maybe list as examples